packages/common/src/validations/workflow-validators/api-plugin.validator.test.ts

```diff
@@ -29,23 +31,23 @@ describe('Api Validator', () => {
 
     describe('when invalid plugin is valid - invalid request', () => {
       it('does not throw validation exception', async () => {
-        let failingPlugin = structuredClone(apiPlugin);
+        const failingPlugin = structuredClone(apiPlugin);
```

Comment:
*non-blocker - replace instances of `structuredClone` in tests with a factory function.